packages/x-codemod/src/v8.0.0/charts/preset-safe/actual.spec.tsx

```diff
@@ -28,4 +28,8 @@ import { ChartsXAxis } from '@mui/x-charts/ChartsXAxis';
   <BarChart slotProps={{ legend: { direction: 'row' } }} />
   <BarChart slotProps={{ legend: { direction: 'column', position: { vertical: 'top', horizontal: 'middle' } } }} />
   <BarChart slotProps={{ legend: { direction: 'wrong' } }} />
+  <BarChart slotProps={{ legend: { position: { vertical: 'middle', horizontal: 'left' } } }} />
```

Comment:
Would be interesting to use this test to make sure the 3 codemod works well together


For example this would test that 

"Legend props propagation" and "Legend position value change" works well together

```suggestion
  <BarChart legend={{ position: { vertical: 'middle', horizontal: 'left' } } } />
```

Reply:
done